src/main/java/uk/gov/hmcts/divorce/idam/IdamService.java

```diff
@@ -37,8 +37,8 @@ public User retrieveUser(String authorisation) {
     public User retrieveSystemUpdateUserDetails() {
         String clusterName = System.getenv().getOrDefault("CLUSTER_NAME", null);
 
-        if (null != clusterName && !clusterName.contains("prod")) {
-            retrieveUser(getCachedIdamOauth2Token(systemUpdateUserName, systemUpdatePassword));
+        if (clusterName == null || !clusterName.contains("prod")) {
```

Comment:
are you sure about this change?

Reply:
I think so? The caching wasn't working on local and I was getting 429s back from Idam - the intended change here was to enable caching for cftLib. Any issues with that

Reply:
why is it turned off in prod? this all just seems a bit odd. Perhaps we should make another PR to remove the if statement

Reply:
Yeah i agree I don't really get it either